src/getHumanAge.test.js

```diff
@@ -7,4 +7,53 @@ describe('getHumanAge', () => {
     expect(getHumanAge)
       .toBeInstanceOf(Function);
   });
+
+  test('should return zeros if catAge and dogAge = 0', () => {
+    const result = getHumanAge(0, 0);
+
+    expect(result)
+      .toEqual([0, 0]);
+  });
+
+  test('should return zeros if catAge and dogAge < 15', () => {
+    const result = getHumanAge(0, 0);
+
+    expect(result)
+      .toEqual([0, 0]);
+  });
+
+  test('should return 1\'s if catAge and dogAge are between 15 and 23', () => {
+    const result = getHumanAge(23, 23);
+
+    expect(result)
+      .toEqual([1, 1]);
+  });
+
+  test('should return first number 2 if catAge are between 24 and 27', () => {
```

Comment:
test cat's and dog's years together "should return 2 for cat and dog when they are 24 years old"